tests/serializers/tests.py

```diff
@@ -278,6 +278,13 @@ def test_deterministic_mapping_ordering(self):
             'second_category_pk': categories[1],
         })
 
+    def test_deserialize_force_insert(self):
+        """Tests that deserialized content can be saved with force_insert as a parameter."""
+        serial_str = serializers.serialize(self.serializer_name, Article.objects.all())
+        models = serializers.deserialize(self.serializer_name, serial_str)
+        for model in models:
+            model.save(force_insert=False)
```

Comment:
This test requires a little bit more validation - all this checks is that force_insert is a valid argument (or, at least, and argument to save() that doesn't raise an error). It doesn't verify that force_insert is passed into the underlying save method.